pkg/telemetry/telemetry.go

```diff
@@ -27,7 +27,11 @@ import (
 // that will be monitored every hour and filled into _ps_catalog.promscale_instance_information table and then into
 // the _timescaledb_catalog.metadata table.
 type Engine interface {
+	// RegisterMetric registers a Prometheus metric with a column name. This metric is
+	// monitored every telemetrySync and updated in the telemetry table.
 	RegisterMetric(columnName string, gaugeOrCounterMetric ...prometheus.Metric) error
+	// Write writes a telemetry information into the database instantly.
+	Write(key, value string)
```

Comment:
Think about propagating error if write fails

Reply:
The child functions do not return error as its telemetry. Hence, no propagation.

Reply:
I think we should at least log an error.